programs/scope-builder.service.js

```diff
@@ -291,10 +292,12 @@ function isInitialVisit(
   });
   // get latest initial encounter and compare with dateenrolled to check whether it's a new enrollment
   let latestEnc = initialEncounters[initialEncounters.length - 1];
+  console.log('has initial', initialEncounters, dateEnrolled);
```

Comment:
Please clean up the console logs
```suggestion
  let latestEnc = initialEncounters[initialEncounters.length - 1];
```